stdlib/experimental/dynamic/dynamic_test.flux

```diff
@@ -6,13 +6,6 @@ import "testing"
 import "experimental/dynamic"
 import "internal/debug"
 
-testcase dynamic_not_comparable {
-    testing.shouldError(
-        fn: () => dynamic.dynamic(v: 123) == dynamic.dynamic(v: 123),
-        want: /unsupported/,
-    )
-}
```

Comment:
With the new `constrain()` implementation, this no longer compiles with `dynamic (dynamic) is not Equatable`. Seems better than catching it at runtime.